src/ResourceManager/Cdn/Commands.Cdn/Models/Profile/PSSkuName.cs

```diff
@@ -20,6 +20,7 @@ public enum PSSkuName
         Premium_Verizon,
         Custom_Verizon,
         Standard_Akamai,
-        Standard_ChinaCdn
+        Standard_ChinaCdn,
```

Comment:
@hytao Does this require any change in the api-version, or does it work in the current api-version?  Can yopu update a test to use the new value?

Reply:
@hytao in addition to the test, you will need to add a change description under `CurrentRelease` in this file : https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/Cdn/Commands.Cdn/ChangeLog.md#current-release

Reply:
Hi @praries880 , I have updated the changelog. Currently I'm having trouble running tests in visual studio. Besides the test, what else should I do? We want to finish this PR ASAP.

Thanks!

Reply:
@hytao  I was able to run the tests on my desktop without any issues.. I ran them in playback mode on a branch which had no changes to the cdn code/tests.